test-suite-http/build.gradle

```diff
@@ -14,7 +14,7 @@ dependencies {
 
     testImplementation(projects.micronautSecurity)
     testImplementation(projects.micronautSecurityJwt)
-    testImplementation(projects.micronautSecurityOauth2)
+    //testImplementation(projects.micronautSecurityOauth2)
```

Comment:
I will investigate this in a separate PR. It seems Security OAuth is coupled to JsonMapper.